src/quantize/tables.rs

```diff
@@ -240,3 +184,24 @@ const ac_qlookup_12_Q3_raw: [u16; QINDEX_RANGE] = [
   22766, 23214, 23662, 24126, 24590, 25070, 25551, 26047, 26559, 27071, 27599,
   28143, 28687, 29247,
 ];
+
+/// # Panics
+///
+/// - If any value is zero
+#[allow(clippy::let_unit_value)]
+#[allow(clippy::unused_unit)]
```

Comment:
These clippy exemptions allow the panic on non-zero below to be formatted as a single line.